packages/radix-ui-themes/src/theme-panel.tsx

```diff
@@ -199,7 +197,6 @@ const ThemePanelImpl = React.forwardRef<ThemePanelImplElement, ThemePanelImplPro
             transitionTimingFunction: open ? 'ease-out' : 'ease-in',
             transform: open ? 'none' : 'translateX(105%)',
             boxShadow: open ? 'var(--shadow-5)' : 'var(--shadow-2)',
-            ...props.style,
```

Comment:
Something to consider here, do we want to support the `style` prop? This was being provided by `Box` I assume.

Reply:
I personally wouldn't remove it and would pass through whatever `div` offers us but it doesn't matter really

Reply:
I removed it since it was showing a type error as we removed the extension of `Box` props but I can adjust the types to pass it through.

Reply:
Whoops. I realized I needed to add the `div` component props after removing `Box`. I'll add this back.